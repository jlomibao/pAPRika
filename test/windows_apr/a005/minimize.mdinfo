

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2744E+02     4.9275E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5264  EEL     =     1332.3596  EGB        =     -127.4416
 1-4 VDW =        5.5061  1-4 EEL =    -2130.7652  RESTRAINT  =        0.0067
 EAMBER  =     -827.4511
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================
