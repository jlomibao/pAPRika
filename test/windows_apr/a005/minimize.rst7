defa
  125
  -0.5278569   4.9585726   1.8302951   1.0085997   4.8941262   1.8312727
  -0.8884754   4.2836435   0.5908342  -0.8942732   4.2462257   3.0467438
  -0.9065637   5.9870732   1.8445700   1.4721589   5.8872583   1.8467391
   1.3132962   4.1919282   0.5921214   1.3127724   4.1528162   3.0475169
   2.6813711   4.0759135   0.1084523   2.6768455   4.0839829   3.5579935
   0.1931118   3.8245031   3.7670919   0.1982438   3.8874712  -0.1420189
   0.1722339   3.3262157   4.8792173   0.1799193   3.4203176  -1.2675638
   2.6662430   4.0299372  -0.9865816   3.2452512   4.9558635   0.4388509
   2.6413324   4.0522841   4.6530623   3.2205322   4.9765359   3.2275417
  -4.6502345   1.9728488   1.8302447  -3.8150944   3.2646894   1.8300153
  -4.2112061   1.2897020   0.6208650  -4.2645501   1.3274906   3.0781603
  -5.7284673   2.1682659   1.8043402  -4.4362560   4.1674937   1.8066154
  -3.0163605   3.1450720   0.6179256  -3.0644638   3.1771977   3.0765513
  -2.2609855   4.2803254   0.1056421  -2.2595255   4.2929044   3.5565593
  -3.3857447   2.0719665   3.8189278  -3.2896268   2.0073919  -0.0952768
  -3.0150719   1.8326217   4.9552592  -2.8548653   1.7262312  -1.1986838
  -2.2485093   4.2306305  -0.9892628  -2.7502675   5.2047953   0.4333531
  -2.2273391   4.2590578   4.6516667  -2.7260532   5.2277480   3.2251022
  -4.0880706  -3.0893362   1.8454972  -4.8094875  -1.7307451   1.8387381
  -3.3252807  -3.0682415   0.6037961  -3.2877753  -3.0355541   3.0614847
  -4.7838430  -3.9360211   1.8669822  -5.9007201  -1.8328132   1.8541776
  -4.3573948  -1.1205002   0.5953832  -4.3241478  -1.0871497   3.0521261
  -4.9307376   0.1283893   0.1137797  -4.9462556   0.1316279   3.5538295
  -3.4691876  -1.8786702   3.7731942  -3.5372850  -1.9364188  -0.1373578
  -3.0176663  -1.6324903   4.8782354  -3.1308012  -1.7253385  -1.2670319
  -4.8869691   0.1375208  -0.9814110  -5.9730464   0.1868320   0.4478747
  -4.9128935   0.1185271   4.6493463  -5.9868433   0.1621225   3.2107701
   0.5804052  -5.1229292   1.8529611  -0.9560583  -5.0576691   1.8538463
   0.9446373  -4.4173461   0.6319946   0.9436679  -4.4384337   3.0864316
   0.9593637  -6.1514026   1.8445405  -1.4208044  -6.0503408   1.8466554
  -1.2608143  -4.3244279   0.6329361  -1.2584480  -4.3441181   3.0873557
  -2.6250811  -4.2525230   0.1253943  -2.6255128  -4.2272700   3.5748656
  -0.1416539  -4.0315672   3.8157030  -0.1425029  -3.9937658  -0.0869280
  -0.1210115  -3.5497601   4.9350017  -0.1223232  -3.4899041  -1.1966123
  -2.5922632  -4.2247926  -0.9698664  -3.1718128  -5.1413211   0.4608917
  -2.6067102  -4.1764997   4.6696306  -3.1892346  -5.1095548   3.2504645
   4.7017953  -2.1343887   1.8374856   3.8678092  -3.4268973   1.8432722
   4.3013275  -1.4859499   0.5958537   4.2738101  -1.4536463   3.0521881
   5.7804920  -2.3284376   1.8514357   4.4892104  -4.3296397   1.8621061
   3.1072112  -3.3387910   0.6031881   3.0766796  -3.3074154   3.0605442
   2.3097549  -4.4593329   0.1234529   2.3160504  -4.4392797   3.5728634
   3.3553961  -2.1706207   3.7731019   3.4124434  -2.2271638  -0.1360476
   2.9260309  -1.8889438   4.8785034   3.0215350  -1.9801840  -1.2639292
   2.2786905  -4.4271107  -0.9717360   2.7801948  -5.3914449   0.4570539
   2.3025899  -4.3888472   4.6677227   2.8024256  -5.3657670   3.2465120
   4.1431949   2.9289176   1.8325271   4.8618843   1.5687035   1.8318733
   3.3398096   2.8809592   0.6185726   3.3860574   2.9050301   3.0780289
   4.8410838   3.7738797   1.8116746   5.9531476   1.6686288   1.8072952
   4.3656628   0.9277505   0.6213454   4.4193383   0.9574294   3.0781154
   4.9807379  -0.2917731   0.1136264   4.9965310  -0.2920240   3.5544126
   3.6055694   1.7735006   3.8178050   3.5099085   1.7232571  -0.0942088
   3.2107144   1.5631298   4.9517062   3.0507289   1.4797089  -1.1966749
   4.9369044  -0.2788980  -0.9815241   6.0245164  -0.3242903   0.4466635
   4.9612300  -0.3021321   4.6499004   6.0362773  -0.3500695   3.2123559
   0.0121610  -0.0762322  -0.0618952  -0.9023136  -0.6611623  -0.2193061
   0.8738901  -0.7358165  -0.2217470   0.0445702   0.7186151  -0.8171118
   0.0389867   0.5179556   1.3576712   0.9474930   1.1195637   1.4837053
  -0.0078633  -0.5844305   2.4351868  -0.8141515   1.1954861   1.4853573
  -0.9168503  -1.1852702   2.3089800   0.8448242  -1.2624576   2.3071432
   0.0197402   0.0080265   3.8553017  -0.8398171   0.6704742   4.0150282
  -0.0160726  -0.7877258   4.6094073   0.9359911   0.5897722   4.0141871
   0.0000573   0.0001435  -6.0006153  -0.0001686  -0.0007856  -9.0000682
   0.0000219   2.2000034 -11.2000229
