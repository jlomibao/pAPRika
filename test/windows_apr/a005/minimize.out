
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 17:56:45

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/a005
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   269295 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -8.1356E+02     1.0057E+01     2.4770E+01     N12        57

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -67.7110  EEL     =     1326.2731  EGB        =     -123.8292
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -813.5649
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.2743E+02     4.9927E-01     1.6814E+00     H7        120

 BOND    =        1.2563  ANGLE   =      107.5586  DIHED      =       52.5864
 VDWAALS =      -68.4329  EEL     =     1332.0623  EGB        =     -127.3910
 1-4 VDW =        5.4805  1-4 EEL =    -2130.5494  RESTRAINT  =        0.0020
 EAMBER  =     -827.4291
 NMR restraints: Bond =    0.001   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.2744E+02     4.9628E-01     1.5809E+00     H32       105

 BOND    =        1.2685  ANGLE   =      107.5457  DIHED      =       52.6105
 VDWAALS =      -68.5135  EEL     =     1332.3302  EGB        =     -127.4487
 1-4 VDW =        5.5014  1-4 EEL =    -2130.7397  RESTRAINT  =        0.0049
 EAMBER  =     -827.4456
 NMR restraints: Bond =    0.001   Angle =     0.004   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.2744E+02     4.9276E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5263  EEL     =     1332.3592  EGB        =     -127.4415
 1-4 VDW =        5.5062  1-4 EEL =    -2130.7649  RESTRAINT  =        0.0067
 EAMBER  =     -827.4510
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.2744E+02     4.9276E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5263  EEL     =     1332.3594  EGB        =     -127.4415
 1-4 VDW =        5.5061  1-4 EEL =    -2130.7651  RESTRAINT  =        0.0067
 EAMBER  =     -827.4511
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.2744E+02     4.9276E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5263  EEL     =     1332.3593  EGB        =     -127.4415
 1-4 VDW =        5.5061  1-4 EEL =    -2130.7650  RESTRAINT  =        0.0067
 EAMBER  =     -827.4510
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.2744E+02     4.9276E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5264  EEL     =     1332.3593  EGB        =     -127.4415
 1-4 VDW =        5.5062  1-4 EEL =    -2130.7650  RESTRAINT  =        0.0067
 EAMBER  =     -827.4510
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.2744E+02     4.9275E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5264  EEL     =     1332.3595  EGB        =     -127.4416
 1-4 VDW =        5.5061  1-4 EEL =    -2130.7653  RESTRAINT  =        0.0067
 EAMBER  =     -827.4511
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.2744E+02     4.9276E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5264  EEL     =     1332.3595  EGB        =     -127.4416
 1-4 VDW =        5.5061  1-4 EEL =    -2130.7652  RESTRAINT  =        0.0067
 EAMBER  =     -827.4511
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.2744E+02     4.9276E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5264  EEL     =     1332.3595  EGB        =     -127.4416
 1-4 VDW =        5.5062  1-4 EEL =    -2130.7651  RESTRAINT  =        0.0067
 EAMBER  =     -827.4510
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2744E+02     4.9275E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5264  EEL     =     1332.3596  EGB        =     -127.4416
 1-4 VDW =        5.5061  1-4 EEL =    -2130.7652  RESTRAINT  =        0.0067
 EAMBER  =     -827.4511
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2744E+02     4.9275E-01     1.5808E+00     H32       105

 BOND    =        1.2607  ANGLE   =      107.5452  DIHED      =       52.6105
 VDWAALS =      -68.5264  EEL     =     1332.3596  EGB        =     -127.4416
 1-4 VDW =        5.5061  1-4 EEL =    -2130.7652  RESTRAINT  =        0.0067
 EAMBER  =     -827.4511
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.302    9.302    0.001    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.339  147.398    0.059    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.482   49.483    0.001    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   61.988   62.003    0.015    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.136 -146.178    0.042    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.654  -72.548    0.106    0.000 t
  DUM  DM1    3 --  C    BUT    2:    5.939    6.000    0.061    0.001 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.237  180.000    0.763    0.001 a
  DUM  DM1    3 --  C3   BUT    2:  178.031  180.000    1.969    0.005 a
                                       Total distance penalty:      0.001
                                       Total angle    penalty:      0.006
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0096
|                               RMS deviation from ideal angles:      4.465
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.32 (22.35% of Gen B)
|                Calc gb diag               4.13 (39.79% of Gen B)
|                Calc gb off-diag           3.93 (37.83% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.38 (100.0% of Nonbo)
|          Nonbond force             10.38 (90.68% of Force)
|          Bond/Angle/Dihedral        1.03 ( 9.02% of Force)
|          Other                      0.03 ( 0.30% of Force)
|       Force time                11.44 (100.0% of Runmd)
|    Runmd Time                11.44 (99.78% of Total)
|    Other                      0.02 ( 0.22% of Total)
| Total time                11.47 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 17:56:45.265  on 04/10/2019
|           Setup done at 17:56:45.271  on 04/10/2019
|           Run   done at 17:56:56.734  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
